docs/built-in-features/storage.md

```diff
@@ -0,0 +1,253 @@
+| DownloadSizeMegabytes | uint64   | Prepaid Download Size in Mb         |
+| FeedbackFeeAmount     | uint64   | XPXs to lock for future payment for |
+| ListOfPublicKeysSize  | uint16   | Size of the list of public keys     |
+| ListOfPublicKeys      | 32 bytes | List of public keys                 |
+
+### DownloadPaymentTransaction
+
+**Version**: 0x01
+
+**Entity type**: 
+
+**Inlines**:
+
+- [Transaction](../protocol/transaction.md#transaction) or [EmbeddedTransaction](../protocol/transaction.md#embeddedtransaction)
+
+| **Property**          | **Type** | **Description**                                                        |
+| --------------------- | -------- | ---------------------------------------------------------------------- |
+| DownloadChannelId     | 32 bytes | The identifier of the download channel                                 |
+| DownloadSizeMegabytes | uint64   | Download size in Mb to add to the prepaid size of the download channel |
+| FeedbackFeeAmount     | uint64   | Amount of XPXs to transfer to the download channel                     |
+
+### FinishDownloadTransaction
+
+**Version**: 0x01
+
+**Entity type**: 
+
+**Inlines**:
+
+- [Transaction](../protocol/transaction.md#transaction) or [EmbeddedTransaction](../protocol/transaction.md#embeddedtransaction)
+
+| **Property**      | **Type** | **Description**                        |
+| ----------------- | -------- | -------------------------------------- |
+| DownloadChannelId | 32 bytes | The identifier of the download channel |
+
+### VerificationPaymentTransaction
+
+**Version**: 0x01
+
+**Entity type**: 
+
+**Inlines**:
+
+- [Transaction](../protocol/transaction.md#transaction) or [EmbeddedTransaction](../protocol/transaction.md#embeddedtransaction)
+
+| **Property**          | **Type** | **Description**                         |
+| --------------------- | -------- | --------------------------------------- |
+| DriveKey              | 32 bytes | Public key of the drive                 |
+| VerificationFeeAmount | uint64   | Amount of XPXs to transfer to the drive |
+
+### EndDriveVerificationTransaction
+
+**Version**: 0x01
+
+**Entity type**: 
+
+**Inlines**:
+
+- [Transaction](../protocol/transaction.md#transaction) or [EmbeddedTransaction](../protocol/transaction.md#embeddedtransaction)
+
+| **Property**              | **Type**                                    | **Description**                                   |
+| ------------------------- | ------------------------------------------- | ------------------------------------------------- |
+| DriveKey                  | 32 bytes                                    | Public key of the drive                           |
+| VerificationTrigger       | 32 bytes                                    | The hash of block that initiated the Verification |
+| ProversCount              | uint16                                      | Number of Provers                                 |
```

Comment:
Are the properties mentioned here meant to match with this?
https://github.com/proximax-storage/cpp-xpx-chain/blob/feature/storage-v3/plugins/txes/storage/src/model/EndDriveVerificationTransaction.h